docs/pages/using_source.md

```diff
@@ -27,7 +27,8 @@ char my_name[SACN_SOURCE_NAME_MAX_LEN];
 // Assuming my_cid and my_name are initialized by the application...
 
 SacnSourceConfig my_config = SACN_SOURCE_CONFIG_DEFAULT_INIT;
-sacn_source_config_init(&my_config, &my_cid, my_name);
+my_config.cid = my_cid;
+memcpy(my_config.name, my_name, SACN_SOURCE_NAME_MAX_LEN);
```

Comment:
Oh, I must have missed this in a previous review - we shouldn't have buffers like this inside of config structs, the name member of SacnSourceConfig should be a const char*